### DISCUSSION TITLE
MinIO License with Supabase

### DISCUSSION THREAD

**Discussion by uahic** [supabase/supabase#29439] [Questions]
Hiho,

I'd like to ask if anyone knows if  Minio (which has an AGPL license) can be used for non-commercial projects without open sourcing my project's source code? Especially if the fact that minio is not directly accessible by the user but routed through supabase components does matter in terms of the license?

In another project I used seaweedfs but the S3 API there is not complete and ultra slow and in general harder to spin up. I used their native APIs in the end (but not in a supabase project, but in a custom spring boot server)

Thank you very much :)

**Comment by silentworks** [comment]
You should try asking this on a Minio platform as they would know more about how the licensing work than someone in this community.
